teams/vsa/onboarding/VSA Technical Orientation.md

```diff
@@ -1,354 +1,489 @@
+
+- **What is a Personal Identity Verification (PIV) Card?**
+
+  - https://www.oit.va.gov/programs/piv
+  - https://www.cac.mil/common-access-card
+  - https://en.wikipedia.org/wiki/Common_Access_Card
+  - https://en.wikipedia.org/wiki/FIPS_201
+
+- **MilitaryCAC<span/>.com** \
+  VA PIVs are essentially the same (hardware and driver-wise) as the DOD Common Access Cards (CACs), so I’ve found https://militarycac.com to be helpful if you have to get down to device/driver troubleshooting details. The site isn’t officially endorsed by the DOD or the VA, but is a community-recognized, grass-roots effort to fill in the gaps left by government help desks. Some of the context is Army/DOD-specific, but if you are technical enough or have the time to sort through those differences, the advice can be applied to VA PIV readers as well.
+  _ https://militarycac.com/macnotes.htm#Mac_friendly
+  _ https://militarycac.com/usbreaders.htm
 
 ## VA Service Desk
+
 The VA National Service Desk (NSD) (aka VA Enterprise Service Desk) number is 855-NSD-HELP (855-673-4357). For this project you might use it for issues with VA Citrix Access Gateway (CAG), VA email, VA Talent Management System (TMS), or outages/downtime associated with VA Enterprise Services.
+
+## Onboarding FAQs
+
+> After I install everything, where do I start?
+
+Answer:
+
+If you are a front-end developer, clone this [website](https://github.com/department-of-veterans-affairs/vets-website) and follow these [set up directions](https://github.com/department-of-veterans-affairs/vets-website/blob/master/README.md). I would try to geth the command `yarn build --buildtype=vagovdev` to work. This will run the app locally and use a dev server as the API. As a bonus, you should try to get the [API](https://github.com/department-of-veterans-affairs/vets-api) set up as well
+
+If you are a back-end developer, clone this [API](https://github.com/department-of-veterans-affairs/vets-api) and get things set up without docker. You might also want to try to get the [website](https://github.com/department-of-veterans-affairs/vets-website) set up as well.
+
+> I want to add a new \_\_\_\_, where do I start?
+
+Checkout out the VA getting start [site](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/). There are two sections on the left, `VA.gov workflow` and `Common tasks`. read through those two sections and try to implement some ideas in a branch.
+
+> Where do I go with my initial technical questions about the site?
+
+First, go to your onboarding buddies, then [slack channels](#slack) that seem appropriate. Ask questions, this the time to really see what is going on.
+
+> Do we have a style guide?
+
+We use prettier and eslint to enforce the Airbnb style guide. We have customized it a little, but usually you editor should be configured to automatically apply those stylings.
```

Comment:
Would like some more detail on "customized it a little". maybe this is missing in VSP's documentation. For now it's good to at least mention it.